pythnet/message_buffer/programs/message_buffer/src/state/message_buffer.rs

```diff
@@ -222,24 +197,33 @@ mod test {
 
         let header_len = MessageBuffer::HEADER_LEN as usize;
 
+        let mut end_offsets = [0u16; u8::MAX as usize];
 
         let (header_bytes, body_bytes) = account_info_data.split_at_mut(header_len);
-        let message_buffer: &mut MessageBuffer = bytemuck::from_bytes_mut(&mut header_bytes[8..]);
+        let _message_buffer: &mut MessageBuffer = bytemuck::from_bytes_mut(&mut header_bytes[8..]);
```

Comment:
if it's unused why not remove it?

Reply:
ill remove this thanks

Reply:
nvm reverting put_all to old implementation so will actually be using this